docs/content/users/extend/customization-extendibility.md

```diff
@@ -109,6 +109,22 @@ web_extra_exposed_ports:
 !!!warning "Fill in all three fields even if you don’t intend to use the `https_port`!"
     If you don’t add `https_port`, then it default to `0` and `ddev-router` will fail to start.
 
+## Exposing Extra Non-HTTP Ports
+
+While the `web_extra_exposed_ports` gracefully handles running multiple ddev projects at the same time, it can't forward  ports for non-HTTP TCP or UDP daemons. Instead, ports can be added in a `docker-compose.*.yaml` file. This file does not need to specify an additional services. For example, this configuration exposes port 5900 for a VNC server.
+
+In `.ddev/docker-compose.vnc.yaml`:
+
+```yaml
+version: '3.6'
+services:
+    web:
+        ports:
+            - "5900:5900"
```

Comment:
Most examples in the docs use two spaces for indentation, so it’s probably best to keep it consistent:

```suggestion
version: '3.6'
services:
  web:
    ports:
      - "5900:5900"
```

Reply:
Good catch, I always use 2 spaces too, so I'm not sure what happened!

Reply:
Just realized this could be a side effect from [this PR](https://github.com/ddev/ddev/pull/4712), if it you were editing locally and it formatted on save. I did some battle with it yesterday.

Reply:
Yep, I have the editorconfig plugin installed in goland. Thanks for pointing out why that happened!

Reply:
Proposal in 
* https://github.com/ddev/ddev/pull/4745

Not sure how we can say indent-4 always in markdown, when it affects fenced codeblocks with yaml.